pavement.py

```diff
@@ -26,9 +28,10 @@ def run(args):
     else:
         jobs = []
         for i in range(4):
-            p = multiprocessing.Process(target=run_behave_test, args=(args[0], "single", i))
+            p = threading.Thread(target=run_behave_test,args=(args[0], "single",i))
             jobs.append(p)
             p.start()
```

Comment:
As we are already maintaining a jobs list, Ideally it should thread_list 😅 

why not follow the best practice to start and join in following:

```suggestion
            p = threading.Thread(target=run_behave_test,args=(args[0], "single",i))
            jobs.append(p)
            p.start()
            
      for th in jobs:
         th.join()
```

Reply:
Start should also be separated from accumulation IMO but depends from user to user.

Reply:
Updated